pkg/provider/azure_loadbalancer_repo.go

```diff
@@ -40,8 +40,7 @@ import (
 
 // DeleteLB invokes az.LoadBalancerClient.Delete with exponential backoff retry
 func (az *Cloud) DeleteLB(service *v1.Service, lbName string) *retry.Error {
-	ctx, cancel := getContextWithCancel()
-	defer cancel()
+	ctx, _ := context.WithTimeout(context.Background(), 60*time.Second)
```

Comment:
why getting the cancel and following `defer cancel()` removed?

Reply:
getContextWithCancel returned a ctx whose parent context is background,  context.WithTimeout did the same thing